src/main/java/com/cryptomorin/xseries/XItemStack.java

```diff
@@ -226,8 +239,43 @@ public static void serialize(@Nonnull ItemStack item, @Nonnull ConfigurationSect
                 fwc.set("fade-colors", fadeColors);
                 i++;
             }
-            //} else if (meta instanceof MapMeta) {
-        } else if (XMaterial.supports(14)) {
+        } else if(meta instanceof BookMeta) {
+            BookMeta book = (BookMeta)meta;
+            ConfigurationSection info = config.createSection("information");
```

Comment:
I don't think they necessarily need to be in an config section. Why not just add `title`, `author` and etc... in the main section?

Reply:
I think it's a good way to organize it, if you don't like it we can change it.

Reply:
Okay, we should use something that's more obvious and shorter like `book`

Reply:
alright np.